web/tests/copy_and_paste.test.js

```diff
@@ -55,6 +57,46 @@ run_test("try_stream_topic_syntax_text", () => {
     }
 });
 
+run_test("latex_math_conversion", () => {
+    const {window} = new JSDOM(`<!DOCTYPE html><html><body></body></html>`);
+    global.document = window.document;
+    global.Element = window.Element;
+    // Test case for inline LaTeX math
+    let input = `
+        <div class="katex-display">
+            <span class="katex-mathml">
+                <annotation encoding="application/x-tex">$$x^2 + y^2 = z^2$$</annotation>
+            </span>
+        </div>`;
+    assert.equal(copy_and_paste.paste_handler_converter(input), "$$x^2 + y^2 = z^2$$");
```

Comment:
Is this the actual HTML for inline LaTeX? I would have thought it'd use spans, not divs.

Reply:
Yes, Sorry a mistake from my side. It's actually span. 
<img width="565" alt="Screenshot 2024-10-04 at 6 01 55 AM" src="https://github.com/user-attachments/assets/4e6eaa5e-ed02-48ba-92d7-df3513b877fa">